[2024-09-03T11:45:11.679+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: k9_etl_dag_v4.check_updates manual__2024-09-03T11:44:59.951661+00:00 [queued]>
[2024-09-03T11:45:11.684+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: k9_etl_dag_v4.check_updates manual__2024-09-03T11:44:59.951661+00:00 [queued]>
[2024-09-03T11:45:11.684+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-09-03T11:45:11.690+0000] {taskinstance.py:1380} INFO - Executing <Task(_PythonDecoratedOperator): check_updates> on 2024-09-03 11:44:59.951661+00:00
[2024-09-03T11:45:11.693+0000] {standard_task_runner.py:57} INFO - Started process 1175 to run task
[2024-09-03T11:45:11.696+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'k9_etl_dag_v4', 'check_updates', 'manual__2024-09-03T11:44:59.951661+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/k9_etl_dag_v4.py', '--cfg-path', '/tmp/tmpab1tolqw']
[2024-09-03T11:45:11.697+0000] {standard_task_runner.py:85} INFO - Job 33: Subtask check_updates
[2024-09-03T11:45:11.721+0000] {task_command.py:415} INFO - Running <TaskInstance: k9_etl_dag_v4.check_updates manual__2024-09-03T11:44:59.951661+00:00 [running]> on host 2624bebdd8ee
[2024-09-03T11:45:11.765+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='k9_etl_dag_v4' AIRFLOW_CTX_TASK_ID='check_updates' AIRFLOW_CTX_EXECUTION_DATE='2024-09-03T11:44:59.951661+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-03T11:44:59.951661+00:00'
[2024-09-03T11:45:11.784+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-09-03T11:45:11.786+0000] {python.py:194} INFO - Done. Returned value was: Found 1 new facts. 1 facts were deleted.
[2024-09-03T11:45:11.800+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=k9_etl_dag_v4, task_id=check_updates, execution_date=20240903T114459, start_date=20240903T114511, end_date=20240903T114511
[2024-09-03T11:45:11.836+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2024-09-03T11:45:11.848+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
